lakefile.lean

```diff
@@ -33,6 +33,7 @@ lean_lib «Doc» where
   -- add library configuration options here
 
 lean_lib «Benchmarks» where
+  leanOptions := #[⟨`weak.benchmark.runs, (0 : Nat)⟩]
```

Comment:
Any documentation for what `weak` is, exactly? I found the [relevant PR here](https://github.com/leanprover/lean4/pull/4741) but I'd rather have a doc. link, if one exists and you're aware of it.

Reply:
I'm not aware of any docs: I got the hint to use `weak` from the error that got thrown when I tried setting `benchmark.runs` directly